server/src/main/java/com/broadcom/lsp/cobol/core/preprocessor/delegates/transformer/ContinuationLineTransformation.java

```diff
@@ -82,12 +83,30 @@ public ContinuationLineTransformation(MessageService messageService) {
   private SyntaxError checkContinuationLine(String uri, int lineNumber, CobolLine cobolLine) {
     if (CobolLineTypeEnum.CONTINUATION.equals(cobolLine.getType())) {
 
+      adjustBlankOrCommentLines(cobolLine);
+
       // invoke method for noContentInAreaA
       return checkContentAreaAWithContinuationLine(cobolLine, uri, lineNumber);
     }
     return null;
   }
 
+  private void adjustBlankOrCommentLines(CobolLine cobolLine) {
+    if (Objects.nonNull(cobolLine.getPredecessor())) {
+      while (isBlankLine(cobolLine.getPredecessor()) || cobolLine.getPredecessor().getType().equals(CobolLineTypeEnum.COMMENT)) {
```

Comment:
It is recommended to use `==` instead of `equals()` for enums to have compile-time type compatibility check

Reply:
done.